labs/lab-2-javascript-game/scripts.js

```diff
@@ -0,0 +1,41 @@
+function startGame() {
+  var start = prompt("Would you like to play a game against the Almighty Grant?");
+  if (start === "yes") {
```

Comment:
If you use `start.toLowerCase()`, here, you can let the user type "yes" or "Yes" (or "YES") which is nice.

(You weren't required to do that, it's just a nice bit of polish you could add, and you'll find it good to know in the future.)